crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/input.sol

```diff
@@ -1 +1,3 @@
-try a.b() {} catch {}
\ No newline at end of file
+// Make sure that error recovery won't lead to misparsing
```

Comment:
I suggest adding another test with call options as well:

```solidity
try foo() {x: 1} {
  bar();
} catch {
}
```

Reply:
Good idea, done in https://github.com/NomicFoundation/slang/pull/862/commits/87b3c786f2e75cde9756cf9db6f1d4e74d3d44fe